shell/creators/pkg/init

```diff
@@ -131,6 +144,25 @@ function fetchLatestVersion() {
           console.log(`  Adding rancher-version annotation '>= ${ release.tag_name }' to package.json`);
 
           pkg.rancher = { annotations: { 'catalog.cattle.io/rancher-version': `>= ${ release.tag_name }` } };
+
+          // read shell version from node_modules/@rancher/shell/package.json
+          const shellPkgRawData = fs.readFileSync(path.join(pkgFolder, `../../${ shellPkgPath }/package.json`));
+          const shellPkg = JSON.parse(shellPkgRawData);
+          const latestShellVersion = shellPkg.version || null;
+
+          if (!latestShellVersion) {
+            console.log('Could not get a shell version from node_modules/@rancher/shell/package.json in app');
+          } else {
+            const splitShellVersion = latestShellVersion.split('.');
+            const majorVersion = splitShellVersion[0];
+            const minorVersion = splitShellVersion[1];
+            const parsedShellVersion = `${majorVersion}.${minorVersion}.0`;
```

Comment:
Sorry if you've already mentioned this, but what's the reasoning behind the patch version always being equal to `0`?

Reply:
Because we should only employ breaking changes on the major version. Feature(s) might be related to minor versions but patch versions should not have any impacts at all, therefore we always start them at `x.x.0`